Sprint-1/exercises/decimal.js

```diff
@@ -1,5 +1,13 @@
 const num = 56.5678;
 
+let wholeNumberPart = Math.floor(num) ;
+let decimalPart = num - Math.trunc(num) ;
+let roundedNum = Math.round(num) ;
+
+console.log(wholeNumberPart)
+console.log(decimalPart)
```

Comment:
Your solution is correct because of round-off error with floating point number arithmetic. However, just for some challenge, can you find a way to output the decimal part of the `num` as .5678? (Currently, it is output as 0.5677999999999983)

Reply:
I fixed it